### ISSUE TITLE
Tree Sticky Scroll - add minimum/maximum for the workbench.tree.stickyScrollMaxItemCount

### ISSUE THREAD

**Issue by lszomoru** [microsoft/vscode#199365]
Testing #199153

In the setting definition all minimum/maximum in order to get input validation in the settings editor.

<img width="896" alt="image" src="https://github.com/microsoft/vscode/assets/3372902/d67f0c47-0520-4c2c-9196-003a0972c2bb">

**Comment by benibenj** [comment]
Good catch. Will set a min value. Don't think a max value is needed though.
